hrm-service/service-tests/case.test.ts

```diff
@@ -328,13 +329,13 @@ describe('/cases route', () => {
                 id: '3',
                 note: 'Child with pneumonia',
                 twilioWorkerId: 'note-adder-1',
-                createdAt: '2022-01-01 00:00:00',
+                createdAt: '2022-01-01T00:00:00+00:00',
```

Comment:
We might possibly want to explicitly convert all incoming dates to UTC? In effect I think that's what would happen in prod anyway, since the servers will all run with a UTC local time, so any dates without explicit TZ info would be treated as UTC anyway.

Explicitly treating dates without a TZ as UTC would prevent any oddities we see in running tests locally in various timezones.

Of course the simplest solution would be to explicitly set UTC as the local time for the HRM node process, but we can't do that, because Nod. :-(